mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/112 camera next previous system (Second!)

Closes #112  This is built on the work in branch issue/160 (#160)

This finishes the impl of the camera next/prev object buttons.

To Test:
- Fire up any scenario
- Mash the next/prev buttons for Users, Computers and Devices
- Add and Scrap computers, ensure the next/prev still work as intended.

Note:
* Lots of code changes due to some namespacing.
* I added a base class for all WorldObjects -- might come in handy down the road when dealing with lists of generic world objects.
